src/context.rs

```diff
@@ -386,25 +392,30 @@ impl BlockOffset {
 #[derive(Copy,Clone)]
 pub struct Block {
     pub mode: PredictionMode,
+    pub bsize: BlockSize,
+    pub partition: PartitionType,
     pub skip: bool,
 }
 
 impl Block {
     pub fn default() -> Block {
         Block {
             mode: PredictionMode::DC_PRED,
+            bsize: BlockSize::BLOCK_64X64,
+            partition: PartitionType::PARTITION_NONE,
             skip: false,
         }
     }
     pub fn is_inter(&self) -> bool {
-        false
+        if self.mode >= PredictionMode::NEARESTMV { true }
+        else { false }
```

Comment:
Could be simplified as:
```
pub fn is_inter(&self) -> bool { 
    self.mode >= PredictionMode::NEARESTMV
}

Reply:
Hey- I saw that and have similar change in my other branch, but the function is not part of this patch, so I believe you could write a patch separately? thx-

Reply:
Oh- sorry, I just realized I touched the function.
I used it as a condition so that prediction is done for each tx block if intra mode and for whole partition if inter mode, but then erased the condition it since it is out of scope of partition task for now.

So, just fixed it any way. Thx!